cmd/protoc-gen-openapi/main.go

```diff
@@ -27,20 +27,21 @@ var flags flag.FlagSet
 
 func main() {
 	conf := generator.Configuration{
-		Version:       flags.String("version", "0.0.1", "version number text, e.g. 1.2.3"),
-		Title:         flags.String("title", "", "name of the API"),
-		Description:   flags.String("description", "", "description of the API"),
-		Naming:        flags.String("naming", "json", `naming convention. Use "proto" for passing names directly from the proto files`),
-		EnumType:      flags.String("enum_type", "integer", `type for enum serialization. Use "string" for string-based serialization`),
-		CircularDepth: flags.Int("depth", 2, "depth of recursion for circular messages"),
+		Version:        flags.String("version", "0.0.1", "version number text, e.g. 1.2.3"),
+		Title:          flags.String("title", "", "name of the API"),
+		Description:    flags.String("description", "", "description of the API"),
+		Naming:         flags.String("naming", "json", `naming convention. Use "proto" for passing names directly from the proto files`),
+		FQSchemaNaming: flags.Bool("fq_schema_naming", false, `schema naming convention. If "true" prefixes the schema name with the proto message package name`),
```

Comment:
Let's spell out what "fq" stands for here in the help string, maybe like this:
```
If "true", generates fully-qualified schema names by prefixing them with the proto message package name.
```

Reply:
I have a couple more PRs coming. I will update the help string there, if that's ok with you.

Reply:
Yes, SGTM